Makefile

```diff
@@ -143,6 +145,9 @@ netmon:
 	bash -f .ci/install_bats.sh
 	bats integration/netmon/netmon_test.bats
 
+tracing:
+	bash tracing/tracing-test.sh
```

Comment:
my brain wants to slap a `chronic` in here ;-)

Reply:
this will only print around 10 lines, so I think we are fine :)